packages/agoric-cli/src/install.js

```diff
@@ -35,14 +41,17 @@ export default async function installMain(progname, rawArgs, powers, opts) {
     p.childProcess.stdout.on('data', out => stdout.push(out));
     await p;
     const d = JSON.parse(Buffer.concat(stdout).toString('utf-8'));
-    const subdirs = Object.values(d).map(v => v.location);
-    return subdirs;
+    Object.entries(d).forEach(([name, { location }]) =>
+      map.set(name, path.resolve(cwd, location)),
+    );
+    return map;
   }
 
   let subdirs;
   const workTrees = ['.'];
   let sdkWorktree;
-  const versions = new Map();
+  /** @type {Map<string, string>} */
+  const sdkPackageToPath = new Map();
```

Comment:
👍 I did have a hunch that `sdkPackageToPath` could be used instead of `isSdkPackage`.